src/std/make.ss

```diff
@@ -101,22 +101,31 @@ TODO:
       optimize: (optimize #t) debug: (debug 'env)
       static: (static #t) static-debug: (static-debug #f)
       verbose: (verbose #f) build-deps: (build-deps_ #f)
-      parallelize: (parallelize_ #t))
+      parallelize: (parallelize_ #f))
     (def gerbil-path (getenv "GERBIL_PATH" "~/.gerbil"))
     (def srcdir (or srcdir_ (error "srcdir must be specified")))
     (def libdir (or libdir_ (path-expand "lib" gerbil-path)))
     (def bindir (or bindir_ (path-expand "bin" gerbil-path)))
     (def prefix (or prefix_ (read-package-prefix srcdir)))
     (def libdir-prefix (if prefix (path-expand prefix libdir) libdir))
     (def build-deps (path-expand (or build-deps_ "build-deps") srcdir))
-    (def parallelize (if (eq? parallelize_ #t) (gerbil-build-cores) (or parallelize_ 0)))
+    (def parallelize (gerbil-build-cores parallelize))
     (struct-instance-init!
       self
       srcdir libdir bindir prefix force? optimize debug static static-debug verbose build-deps
       libdir-prefix parallelize)))
 
-(def (gerbil-build-cores)
-  (with-catch (lambda (_) (##cpu-count)) (lambda () (string->number (getenv "GERBIL_BUILD_CORES")))))
+(def (gerbil-build-cores (cpu-count-spec #t))
+  ;; TODO: for the default (catch) case, use something like
+  ;; (min (##cpu-count) (/ (available-memory) (memory-per-compilation-cpu)))
+  ;; Except we need to compute (available-memory) and calibrate (memory-per-compilation-cpu)...
+  ;; Until then, we disable the parallel build as it confuses a lot of beginners.
+  (cond
+   ((real? cpu-count-spec) cpu-count-spec)
```

Comment:
probably fixnum? or integer? is a better test.

Reply:
All I do with the object is decrement and compare to zero, so as long as it's an archimedean ring... and actually, I suppose `+inf.0` means no limit.

Reply:
```
> (integer? 2.e53)
#t
> (= (+ 1 2.e53) 2.e53)
#t
```

Reply:
uhm, it would probably be a bad idea --  do we want to spawn a 1000 gccs?

Reply:
So `2**53-1` as a fixnum is A-OK, but `2**53-1` as a float isn't? Not that either will make much of a difference, unless you have that many processes.

Reply:
yeah well, I guess if the user wants to be silly and lock up their machine, it's up to them.

Reply:
feel free to merge, this is not blocking :)